ui/raidboss/data/07-dt/raid/r1s.ts

```diff
@@ -217,6 +217,7 @@ const triggerSet: TriggerSet<Data> = {
       id: 'R1S Headmarker Nailchipper Spread',
       type: 'HeadMarker',
       netRegex: { id: headMarkerData.spreadMarker1, capture: false },
+      condition: Conditions.targetIsYou(),
       suppressSeconds: 5,
```

Comment:
How does this interact with the condition target is you? Will it only suppress after the condition is true? I worry that might accidentally suppress this signal entirely if it doesn't work that way..

Reply:
https://github.com/OverlayPlugin/cactbot/blob/main/docs/RaidbossGuide.md#trigger-properties

Reply:
Ah thanks for the doc :)